{
  "comments": [
    {
      "key": {
        "uuid": "054b7b9e_6ec47fff",
        "filename": "INFO.yaml",
        "patchSetId": 6
      },
      "lineNbr": 3,
      "author": {
        "id": 481
      },
      "writtenOn": "2021-01-07T16:57:41Z",
      "side": 1,
      "message": "This is wrong... it should be 2020-12-03",
      "range": {
        "startLine": 3,
        "startChar": 23,
        "endLine": 3,
        "endChar": 35
      },
      "revId": "f1841463cd3431e699dfa201286f42a4018087ff",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    }
  ]
}